src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.Cart/EditOrder/EditOrderBannerController.ts

```diff
@@ -32,6 +32,15 @@ module Orckestra.Composer {
                 methods: {
                     cancelEditOrder() {
                         self.orderService.cancelEditOrder(this.OrderNumberForOrderDraft);
+                    },
+                    updateEditOrder() {
```

Comment:
is this logic needed? you can heave just the correct href="url" when you render Update Order link in banner